### ISSUE TITLE
Can't add device

### ISSUE THREAD

**Issue by casper181** [jordenc/com.synology.surveillance#13]
I enter the NAS details, click save then nothing. What should I expect?

**Comment by casper181** [comment]
![Screenshot_20190401-082800_Homey](https://user-images.githubusercontent.com/6625054/55310367-36843500-5458-11e9-95eb-b80d396aa537.jpg)

I complete this screen and hit Save settings then nothing

**Comment by jordenc** [comment]
What did you fill in (don't need the username/password).

What often goes wrong, is that people add http:// to the IP, or they don't fill in the port or username if it's 500/admin.
Please also note that the username/password are case sensitive and make sure you read all requirements on the Github / app store page.

**Comment by casper181** [comment]
IP address  192.168.1.173
Port: 5837 (port of the Diskstation)
Username: <hidden>
Password: <hidden>

Once I hit the Save settings button it says 'Saved!' above the '_Please make sure......_'

**Comment by jordenc** [comment]
That should be fine I guess. As long as Homey is in the same home network as your Synology.

Does the username have "Manager" rights?

(from the readme:
Login to your Synology
Click on "Surveillance Station" so it opens in a new window.
Click on the menu button, go to "User".
Click on "Add", fill in a username and password (remember both, you will be neeeing them later)
Click "Next", make sure to choose the profile with "Manager" rights.
Click "Next", make sure to check all cameras you want to give access to.
Click "Complete".)

**Comment by casper181** [comment]
Yes, definitely has manager rights. After getting 'Saved!' should I expect the screen to change and take me somewhere?

**Comment by casper181** [comment]
Cracked it, it needed a different Port number (not the Synology itself)

**Comment by jordenc** [comment]
Good to hear! What was the difference? (Other users may struggle with the same problem)

**Comment by casper181** [comment]
On the Synology NAS I went to Control Panel -> External Access -> Router Configuration -> Create

Then I used the first Surveillance Station port in the list and that worked. I didn't actually change any settings here as they're both on the same network so I didn't need to configure port forwarding

**Comment by Lusch81** [comment]
I have the same problem. I tried my DiskStation port (5555), but also the standard Surveillance Station ports (554, 19998). I also tried with a new manager user created in Surveillance Station settings.

I keep get the message `Saved!` without it acutally adding the DiskStation a a device.

**Comment by loki679** [comment]
Same here. Tried all what was posted before but nothing works. Get only the saved message and nothing more happens.

**Comment by A333333** [comment]
Same here too! I appreciate all your hard work but:
Tried everything for weeks now but cannot add any SS or camera. Tried several ports, like 5000, 5001, 9900, 9901 but nothing after the 'Saved!' message. I am using the v2.0.3 app on Homey Pro latest firmware. I am on the local network, looked into the Synology Firewall, double checked the added account for Homey SS but..... nothing… It is not clear to me if this app is supported on Homey Pro and if HTTPS is supported (My synology redirect to HTTPS if you try to connect over HTTP) Cannot find this info in the relase notes.
I've used your app on Homey 1 before FW 2.0, I liked it a lot, and I want to make it work but now you have to make clear what is supported on which Homey (1 or Pro) on which firmware. Looking for a needle in a haystack here.
